src/librbd/cache/rwl/LogEntry.cc

```diff
@@ -0,0 +1,43 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include <iostream>
+#include "LogEntry.h"
+
+#define dout_subsys ceph_subsys_rbd
+#undef dout_prefix
+#define dout_prefix *_dout << "librbd::cache::rwl::LogEntry: " << this << " " \
+			   <<  __func__ << ": "
+
+namespace librbd {
+
+namespace cache {
+
+namespace rwl {
+
+bool GenericLogEntry::is_sync_point() { return ram_entry.is_sync_point(); }
```

Comment:
Nit: wrap `return` down to next line (applies throughout)

Reply:
Done